crates/rpc-proto/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "miden-rpc-proto"
+version = "0.3.0"
+description = "Miden RPC message definitions"
+readme = "README.md"
+keywords = ["miden", "node", "protobuf", "rpc"]
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[features]
+default = ["std"]
+std = []
+
+[dependencies]
+hex = { version = "0.4" }
+miden-node-utils = { workspace = true }
+miden-objects = { workspace = true }
+prost = { version = "0.12" }
+thiserror = { workspace = true }
+tonic = { workspace = true }
+
+[build-dependencies]
+miette = { version = "7.0", features = ["fancy"] }
+prost = { version = "0.12" }
+prost-build = { version = "0.12" }
+protox = { version = "0.6" }
+tonic-build = { version = "0.11" }
```

Comment:
Do we also need all of these build dependencies?

Reply:
Removed these